js/cards.js

```diff
@@ -85,15 +96,41 @@ export async function defineCardComponent() {
       return `${CLOUDINARY_URL}/q_auto/${this.data["Rarity Folder"]}/${this.data["Filename"]}.png`;
     }
 
+    resetCard() {
+      card_count = 10;
+      this.back.classList.remove("hidden");
+      this.front.tabIndex = "-1";
+
+      addHover();
+    }
+
     flipCard() {
-      this.front.classList.toggle("hidden");
-      this.back.classList.toggle("hidden");
+
```

Comment:
(nit) took me a minute to understand this code - a comment like "remove event listeners from previous flipped card and flip over + set up event listeners for current card" would help

Reply:
The particular code here is no longer present, but I have added some comments to explain what is happening.